### PR TITLE
CLDR-17975 MessageFormat v46 Technical Preview pour

### PR DESCRIPTION
**Pull request by aphillips**
CLDR-17975

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: common/dtd/messageFormat/message.dtd
Comment (aphillips) [inline common/dtd/messageFormat/message.dtd:1]:
```suggestion
<!--
Copyright © 1991-2024 Unicode, Inc.
For terms of use, see http://www.unicode.org/copyright.html
SPDX-License-Identifier: Unicode-3.0
CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
-->

<!ELEMENT message {
```

File: common/dtd/messageFormat/message.dtd
Comment (aphillips) [inline common/dtd/messageFormat/message.dtd:8]:
```suggestion
<!ELEMENT message (
```

File: common/dtd/messageFormat/message.json
Comment (aphillips) [inline common/dtd/messageFormat/message.json:3]:
```suggestion
"$id": "https://github.com/unicode-org/cldr/blob/maint/maint-46/common/dtd/messageFormat/message.json",
```

File: common/testData/messageFormat/README.md
Comment (aphillips) [inline common/testData/messageFormat/README.md:3]:
```suggestion
For information about MessageFormat 2.0, see [Unicode Locale Data Markup Language (LDML): Part 9: Message Format](../../../docs/ldml/tr35-messageFormat.md)
```

**Review by macchiati — APPROVED** [review]

**Comment by macchiati** [comment]
Ack, the commit messages were wrong.

Each one has to start with 
CLDR-17975<space>

The easiest way to fix at this point is to click on Details under "jira-ticket" above. (Direct link https://us-central1-dev-infra-273822.cloudfunctions.net/unicode-github-bot/info/unicode-org/cldr/4078). Hit [Squash Branch] which will result in a new commit message (which will start with CLDR-17975<space>), and click " Yes, I understand. This is what I want." and [Perform Squash].

Let me know when this is done, and I'll ok it again if needed, and Merge.

**Comment by macchiati** [comment]
Looks great. I'm at the doctor's, will merge once the checks finish. Thanks

CC @AEApple

**Comment by aphillips** [comment]
@macchiati I think we got it now. Thanks for the hint.

Note that we'd like it if the [linkifier](https://github.com/unicode-org/message-format-wg/blob/main/docs/tools/linkify.js) were run on the final HTML. But that's not *required* just now.

**Comment by AEApple** [comment]
I'll merge since all tests have passed.

**Comment by macchiati** [comment]
I agree. I'd recently filed
https://unicode-org.atlassian.net/browse/CLDR-17970. Ideally all spec and
site pages would have to have no broken links before they could be merged.
(or at least no broken links to other Unicode pages.

**Comment by aphillips** [comment]
@macchiati Note that the linkifier *adds* links inside our spec from terms to the definition. E.g. if it says _message_ a link gets added to the `**_<dfn>message</dfn>_**` location in the spec. Other Unicode specs might benefit from this (we do a lot of auto-term linking in W3C via our tools Respec or bikeshed

**Comment by macchiati** [comment]
Ah, that would be very useful for the whole LDML spec (and others) We already process for the TOC, this would be a useful addition. cc @loomis

**Comment by macchiati** [comment]
I filed https://unicode-org.atlassian.net/browse/CLDR-18029 
cc @srl295

**Pull request merged**
